nx/test/nx/defn/grad_test.exs

```diff
@@ -478,6 +478,22 @@ defmodule Nx.Defn.GradTest do
     end
   end
 
+  describe "inverse trig family" do
+    defn grad_arcsin(t), do: grad(t, Nx.arcsin(t))
+    defn grad_arccos(t), do: grad(t, Nx.arccos(t))
+    defn grad_arctan(t), do: grad(t, Nx.arctan(t))
+
+    test "computes gradient of inverse trig functions" do
+      for _ <- 1..100 do
+        t = Nx.random_uniform({}, -0.996, 0.996, type: {:f, 64})
```

Comment:
@seanmor5 I added the changes in the grad functions for `arccos` and `arcsin` but now the tests only work up to plus/minus 0.996. I see the benefit of having the more readable gradient though.